nextjs-starter/baml_src/semantic_streaming.baml

```diff
@@ -0,0 +1,33 @@
+class SemanticContainer {
```

Comment:
can we update the recipie example to use semantic streaming to make the example the same then we can see how much code we deleted.

additionally, i would check which things we can clean up in general in the type system using semantic streaming.

like perhaps the resume example?
the book example?

this would give us nicer coverage on all of this. but already looks really nice!